environment.yml

```diff
@@ -17,6 +17,9 @@ dependencies:
   - seaborn
   - matplotlib
   - geopandas
+  - pandas
+  - geoviews
+  - jupyterthemes
```

Comment:
Neither `geoviews` or `jupyterthemes` are required by the code - we should remove.